apps/frontend/src/libs/components/header/header.tsx

```diff
@@ -0,0 +1,17 @@
+import { Avatar, Logo } from "~/libs/components/components.js";
+import { getValidClassNames } from "~/libs/helpers/helpers.js";
+
+import styles from "./styles.module.css";
+
+const Header: React.FC = () => {
+	return (
+		<header className={getValidClassNames(styles["header"])}>
```

Comment:
There is no need to wrap classname in `getValidClassNames` as you don't need to combine a couple of classnames
```suggestion
		<header className={styles["header"]}>
```